upf/upf_cli.c

```diff
@@ -306,7 +306,10 @@ upf_nat_pool_add_del_command_fn (vlib_main_t * vm,
   int rv;
 
   if (!unformat_user (main_input, unformat_line_input, line_input))
-    return 0;
+    {
+      error = 0;
+      goto done;
```

Comment:
In this particular case, keeping `return 0` is better, as otherwise the code will crash (see below)